packages/typescript/src/components/Prose.tsx

```diff
@@ -1,15 +1,15 @@
 import { childrenArray, computed } from "@alloy-js/core";
 import { Children } from "@alloy-js/core/jsx-runtime";
 
-export interface JSDocParagraphProps {
+export interface Prose {
   children: Children;
 }
 
 /**
- * Create a paragraph in a JSDoc comment. The text of this component is word
- * wrapped.
+ * Creates a prose element. This is a special element that breaks the children into
```

Comment:
Create a block of text which will break once a word exceeds the configured line width. The children are expected to be strings, and a `<br />` is added between each word.